tests/Backend/Synapse/TableDefinitionOperationsTest.php

```diff
@@ -59,30 +59,30 @@ private function createTableDefinition()
 
     public function testAddColumnOnTypedTable()
     {
-        $this->expectExceptionMessage("Not implemented for typed synapse tables");
+        $this->expectExceptionMessage("Not implemented for typed tables");
         $this->_client->addTableColumn($this->tableId, 'newColumn');
     }
 
     public function testDropColumnOnTypedTable()
     {
-        $this->expectExceptionMessage("Not implemented for typed synapse tables");
+        $this->expectExceptionMessage("Not implemented for typed tables");
         $this->_client->deleteTableColumn($this->tableId, 'name');
     }
 
     public function testAddPrimaryKeyOnTypedTable()
     {
-        $this->expectExceptionMessage("Not implemented for typed synapse tables");
+        $this->expectExceptionMessage("Not implemented for typed tables");
         $this->_client->createTablePrimaryKey($this->tableId, ['id']);
     }
     public function testRemovePrimaryKeyOnTypedTable()
     {
-        $this->expectExceptionMessage("Not implemented for typed synapse tables");
+        $this->expectExceptionMessage("Not implemented for typed tables");
         $this->_client->removeTablePrimaryKey($this->tableId);
     }
 
     public function testCreateSnapshotOnTypedTable()
     {
-        $this->expectExceptionMessage("Not implemented for typed synapse tables");
```

Comment:
trochu BC break, ale asi naprosto minimalni

Reply:
To bych asi vůbec neřešil je to experimental a ještě not implemented string message